src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -64,6 +65,42 @@ fn datablocks_to_sectors(data_blocks: DataBlocks) -> Sectors {
     *data_blocks * DATA_BLOCK_SIZE
 }
 
+pub fn get_dirty_page_size() -> StratisResult<Sectors> {
```

Comment:
Name quibbles: I'll spare you the anguished details, but how about we just call this `current_dirty_mem`?

Reply:
Ok.  Thanks.